generator:
  theta_ld_min: -1.57 # -1.57 rad. Also -90 deg.
  theta_ld_max: 1.57 # 1.57 rad. Also 90 deg.
  theta_ld_resolution: 0.002 # 0.002 rad. Around 0.1 deg of resolution.
  theta_l_min: -1.57 # -1.57 rad. Also -90 deg.
  theta_l_max: 1.57 # 1.57 rad. Also 90 deg.
  theta_l_resolution: 0.002 # 0.002 rad. Around 0.1 deg of resolution.
  theta_m_min: -1.57 # -1.57 rad. Also -90 deg.
  theta_m_max: 1.57 # 1.57 rad. Also 90 deg.
  theta_m_resolution: 0.002 # 0.002 rad. Around 0.1 deg of resolution.
  theta_m_p_min: 1.04 # In rd/s. In RPM it would be 10 RPM (choice of a min rotational speed)
  theta_m_p_max: 8.06 # In rd/s. In RPM it would be 77 RPM (Dynamixel max rotational speed)
  theta_m_p_resolution: 0.02 # 0.02 rad/s. Around 1 deg/s of resolution.
